she/promote.py

```
@@ -183,6 +183,10 @@ def plan_promotion(
         )
 
     verification = verification or plan_verification(incident)
+    if verification.incident_id != incident.incident_id:
+        raise PromotionError("verification incident_id must match incident")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)